app/components/task-selection/component.js

```diff
@@ -237,13 +240,15 @@ export default Component.extend({
       return value
     }
   },
+  /* eslint-enable ember/avoid-leaking-state-in-ember-objects */
```

Comment:
Hmm, the setters in computed properties are really not too nice, right? Did you consider refactoring this?

Reply:
BTW: Reason for this ignore thingy: https://github.com/ember-cli/eslint-plugin-ember/issues/42